include/string_util.h

```diff
@@ -0,0 +1,17 @@
+#ifndef POKEPLATINUM_STRING_UTIL_H
+#define POKEPLATINUM_STRING_UTIL_H
+
+
+#include <nnsys.h>
+
+#define MAX_STRING_COPY_LEN    256
+#define ASCII_LINE_FEED        0xA
+#define ASCII_CARRIAGE_RETURN  0xD
```

Comment:
[question] Why not just use `'\n'` and `'\r'`?

Reply:
I wish I could say I had a good reason :)
fixed